### PR TITLE
Fix sanitization on FormTextItem

### PR DESCRIPTION
**Pull request by nauaros**
## Summary
Fix for the missing sanitization on the `FormTextItem`

### CODE REVIEW

File: Tests/AdyenTests/Adyen Tests/UI/Form/Items/Text/FormTextItemViewTests.swift [hunk 1]
```diff
@@ -169,20 +169,25 @@ class FormTextItemViewTests: XCTestCase {
...
+        let expectedFormattedValue = formatter.formattedValue(for: expectedItemValue)
+
+        // When
+        sut.textField.text = "2222XF%4000"
         sut.textField.sendActions(for: .editingChanged)
-        wait(for: .seconds(1))
 
-        XCTAssertEqual(sut.textField.text, "541")
+        // Then
+        XCTAssertEqual(expectedItemValue, item.value)
```
Comment (mohammedDehairy) [inline Tests/AdyenTests/Adyen Tests/UI/Form/Items/Text/FormTextItemViewTests.swift:188]: that's actually not correct, the `item.value` should be the sanitized value, but not formatted.

File: Tests/AdyenTests/Card Tests/Items/CardNumberItem/FormCardNumberItemViewTests.swift [hunk 2]
```diff
@@ -80,4 +80,50 @@ class FormCardNumberItemViewTests: XCTestCase {
...
+        // Then
+        let expectedItemValue = "5555341244441115"
+        XCTAssertEqual(expectedItemValue, sut.item.value)
+
+        let expectedItemFormattedValue = "5555 3412 4444 1115"
+        XCTAssertEqual(expectedItemFormattedValue, sut.item.formattedValue)
+
+        let expectedTextFieldText = sut.item.formattedValue
+        XCTAssertEqual(expectedTextFieldText, sut.textField.text)
+    }
```
Comment (mohammedDehairy) [inline Tests/AdyenTests/Card Tests/Items/CardNumberItem/FormCardNumberItemViewTests.swift:128]: this is nice !

File: Tests/AdyenTests/Adyen Tests/UI/Form/Items/Text/FormTextItemViewTests.swift (hunk 1 above)
Comment (nauaros) [reply Tests/AdyenTests/Adyen Tests/UI/Form/Items/Text/FormTextItemViewTests.swift:188]: Here `expectedItemValue` is the sanitized value, there's no formatting in "22224000"

Comment (mohammedDehairy) [reply Tests/AdyenTests/Adyen Tests/UI/Form/Items/Text/FormTextItemViewTests.swift:188]: yeah, its coincidence that sanitized is actually equal to formatted, ok, no problem then.

File: Adyen.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2028,14 +2048,13 @@
...
 			isa = PBXGroup;
 			children = (
-				E7D3ED83243B667500E73F59 /* FormCardNumberItemViewTests.swift */,
-				E2B317AB22660FB600C1BB30 /* FormCardNumberItemTests.swift */,
-				F9A781AD2403BD1600E12487 /* FormItemViewBuilderTests.swift */,
+				C978F5EC2732CD0800F59B3C /* CardNumberItem */,
+				C978F5ED2732CD4700F59B3C /* CardSecurityCodeItem */,
 			);
-			path = Form;
+			path = Items;
```
Comment (descorp) [inline Adyen.xcodeproj/project.pbxproj:2057]: I believe `Form` was more descriptive then `Items` 🤔

Comment (nauaros) [reply Adyen.xcodeproj/project.pbxproj:2057]: `FormItems` maybe. I'd like to emphasize that item related stuff is being tested within the group

Comment (descorp) [reply Adyen.xcodeproj/project.pbxproj:2057]: Why not :)
I just thing there should be "Form" in the name somewhere

**Review by mohammedDehairy — APPROVED** [review]

**Review by mohammedDehairy — APPROVED** [review]

**Review by descorp — APPROVED** [review]

**Pull request merged**